payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherViewModel.kt

```diff
@@ -218,7 +221,12 @@ internal class GooglePayLauncherViewModel(
                 paymentController.getSetupIntentResult(data)
             }
             else -> {
-                Result.failure(IllegalStateException("Unexpected confirmation result."))
+                val error = IllegalStateException("Unexpected confirmation result.")
+                errorReporter.report(
```

Comment:
Is there any more data that might be useful to send up in this scenario as nonPiiParams?

Reply:
Added requestCode -- the implementations of `paymentController.shouldHandleSetupResult` and `paymentController.shouldHandlePaymentResult` are basically just checking the request code. So if we get to this branch, we must have a different request code that we are not handling